### ISSUE TITLE
Trailing spaces not allowed  no-trailing-spaces

### ISSUE THREAD

**Issue by athifrank** [eslint/eslint#7791]
```js
import React, { Component } from 'react';
import { ListGroupItem } from 'react-bootstrap';
import { Link } from 'react-router';

class ContactListItem extends Component {
  render() {
    const { contact } = this.props;
    return (
      <ListGroupItem>
        <Link to={'/contact/${contact.id}'}>      ======>this is issue line    
          <h4>{contact.name}</h4>
        </Link>
      </ListGroupItem>
    );
  }
}
```

---------------------------------------------------------------------------

WARNING in ./src/components/ContactListItem.js

C:\react-sites\react-auth0-master\src\components\ContactListItem.js
  10:45  warning  Trailing spaces not allowed  no-trailing-spaces

✖ 1 problem (0 errors, 1 warning)

webpack: bundle is now VALID.

**Comment by ljharb** [comment]
@athifrank i'm not sure what the issue is - does that line in fact not have trailing spaces?

**Comment by eslintbot** [comment]
Hi @athifrank, thanks for the issue. It looks like there's not enough information for us to know how to help you.

If you're reporting a bug, please be sure to include:

1. The version of ESLint you are using (run `eslint -v`)
2. What you did (the source code and ESLint configuration)
3. The actual ESLint output complete with numbers
4. What you expected to happen instead

Requesting a new rule? Please see [Proposing a New Rule](http://eslint.org/docs/developer-guide/contributing/new-rules) for instructions.

Requesting a rule change? Please see [Proposing a Rule Change](http://eslint.org/docs/developer-guide/contributing/rule-changes) for instructions.

If it's something else, please just provide as much additional information as possible. Thanks!

**Comment by nzakas** [comment]
No response and not enough detail, so closing.
